examples/auth-with-stytch/.env.template

```diff
@@ -1,8 +1,8 @@
 # The two iron-session values may be left as-is while testing out this example app.
-IRON_SESSION_COOKIE_NAME="stytch_next_example_cookie"
-IRON_SESSION_PASSWORD="complex_password_at_least_32_characters_long"
+NEXT_PUBLIC_IRON_SESSION_COOKIE_NAME="stytch_next_example_cookie"
+NEXT_PUBLIC_IRON_SESSION_PASSWORD="complex_password_at_least_32_characters_long"
 # The below values may be found in your Stytch Dashboard: https://stytch.com/dashboard/api-keys
-STYTCH_PROJECT_ENV=test
-STYTCH_PROJECT_ID="YOUR_STYTCH_PROJECT_ID"
-STYTCH_SECRET="YOUR_STYTCH_SECRET"
-STYTCH_PUBLIC_TOKEN="YOUR_STYTCH_PUBLIC_TOKEN"
+NEXT_PUBLIC_STYTCH_PROJECT_ENV=test
+NEXT_PUBLIC_STYTCH_PROJECT_ID="YOUR_STYTCH_PROJECT_ID"
+NEXT_PUBLIC_STYTCH_SECRET="YOUR_STYTCH_SECRET"
```

Comment:
If the env var has `NEXT_PUBLIC_` as a prefix, it will be exposed on the client side. I'm guessing you want to keep these secrets server-side, correct?

Reply:
Thanks so much for catching this @leerob! These were here from my testing and I made the critical error of committing at midnight after "it was working". Removed in 856b26b.